.gitignore

```diff
@@ -8,3 +8,5 @@
 *.iml
 .idea/
 target/
+
+lambda$*.json
```

Comment:
What's this for?

Reply:
I think this is related to running the snapshot related unit tests under Windows. Instead of using:

src\test\resources\snapshots\com\conveyal\gtfs\graphql\GTFSGraphQLTest\canFetchAgencies-0.json

the following is created instead:

src\test\resources\snapshots\com\conveyal\gtfs\graphql\GTFSGraphQLTest\lambda$canFetchAgencies$7-0.json

This is repeated for all 22 snapshot unit tests. I think it is related to how the paths are interpreted in Windows and maybe similar to a fix that was put in place for otp-middleware. 

For now, I have added this to git ignore so I don't accidentally commit them. Once I have addressed I will remove.